unuran/src/methods/mvtdr_init.h

```diff
@@ -1757,7 +1757,7 @@ _unur_mvtdr_etable_new( struct unur_gen *gen, int size )
   GEN->etable_size = size;
 
   /* make root */
-  GEN->etable = malloc( size * sizeof(E_TABLE*) );
+  GEN->etable = malloc( size * sizeof(E_TABLE) );
```

Comment:
This has been changed in unuran 1.9.0. See #2.

`GEN->etable` is a `E_TABLE **` so the change here doesn't seem like the correct fix. The warnings thrown by GCC seems to be because of an implicit type conversion issue. Can you please try to build again with changes in #2 and check if the warning persists?